scripts/travis/install-requirements.sh

```diff
@@ -3,6 +3,11 @@
 if [ "${TASK}" = 'compilepy3 ci-py3-unit' ] || [ "${TASK}" = 'ci-py3-integration' ]; then
     pip install "tox==3.8.6"
 
+	# cleanup any invalid python2 cache
```

Comment:
Whitespace issue here - use of tabs, when the rest of the diff uses spaces.

Reply:
oops. will fix